x/crosschain/keeper/msg_server_refund_aborted_tx.go

```diff
@@ -0,0 +1,93 @@
+package keeper
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/x/crosschain/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+	"golang.org/x/net/context"
+)
+
+func (k msgServer) RefundAbortedCCTX(goCtx context.Context, msg *types.MsgRefundAbortedCCTX) (*types.MsgRefundAbortedCCTXResponse, error) {
```

Comment:
Can we please add function documentation

Reply:
ca87db9d1d9d2836ccb297bad1ad91497ce448d9